### PR TITLE
ELECTRON-101 - Electron build issues

### PR DESCRIPTION
**Pull request by KiranNiranjan**
1. Moved spectron test to a different directory
2. Added a `npm` command to run spectron test separately

@VikasShashidhar & @VishwasShashidhar Please review. Thanks

### CODE REVIEW

File: tests/activityDetection.test.js [hunk 1]
```diff
@@ -3,10 +3,10 @@ const childProcess = require('child_process');
 
 let activityDetection;
 
-describe('Tests for Activity Detection', function() {
+describe('Tests for Activity Detection', function () {
 
     var originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 90000;
```
Comment (VishwasShashidhar) [inline tests/activityDetection.test.js]: Is there no way to do this dynamically? What if we run this on a low configuration system?

File: package.json [hunk 2]
```diff
@@ -15,7 +15,8 @@
     "prebuild": "npm run rebuild && npm run browserify-preload",
     "browserify-preload": "browserify -o js/preload/_preloadMain.js -x electron --insert-global-vars=__filename,__dirname js/preload/preloadMain.js",
     "rebuild": "electron-rebuild -f",
-    "test": "npm run lint && jest --verbose --testPathPattern test --runInBand",
+    "test": "npm run lint && jest --verbose --testPathPattern test && npm run rebuild && npm run spectron-test",
+    "spectron-test": "jest --config tests/spectron/jest_spectron.json --runInBand && npm run rebuild",
```
Comment (VishwasShashidhar) [inline package.json]: We should have a script that runs both the spectron and unit tests one after the other.

File: tests/spectron/notificationPosition.spectron.js [hunk 3]
```diff
@@ -45,9 +45,9 @@ describe('Tests for Notification position', () => {
     it('should load demo html page', () => {
         let filePath;
         if (process.platform === 'win32') {
-            filePath = 'file:///' + path.join(__dirname, '..', 'demo/index.html');
+            filePath = 'file:///' + path.join(__dirname, '..', '..', 'demo/index.html');
```
Comment (VishwasShashidhar) [inline tests/spectron/notificationPosition.spectron.js]: Can we just do `path.join(__dirname, '../../', 'demo/index.html'? Won't that work?

File: tests/spectron/notificationPosition.spectron.js [hunk 4]
```diff
@@ -45,9 +45,9 @@ describe('Tests for Notification position', () => {
...
         } else {
-            filePath = 'file://$(pwd)/' + path.join(__dirname, '..', 'demo/index.html')
+            filePath = 'file://$(pwd)/' + path.join(__dirname, '..', '..', 'demo/index.html')
```
Comment (VishwasShashidhar) [inline tests/spectron/notificationPosition.spectron.js]: Can we just do `path.join(__dirname, '../../', 'demo/index.html'? Won't that work?

File: package.json (hunk 2 above)
Comment (KiranNiranjan) [inline package.json]: `npm run test` will first run the unit tests and then spectron with `npm run spectron-test`

File: tests/spectron/notificationPosition.spectron.js (hunk 3 above)
Comment (KiranNiranjan) [inline tests/spectron/notificationPosition.spectron.js]: Done 👍

File: tests/activityDetection.test.js (hunk 1 above)
Comment (VishwasShashidhar) [inline tests/activityDetection.test.js]: I'll approve this for now. We'll need to find a soln. though. Please create a new PR once you find a fix.

**Comment by VishwasShashidhar** [comment]
lgtm

**Comment by VikasShashidhar** [comment]
LGTM 👍

**Pull request merged**